finance.adoc

```diff
@@ -120,25 +149,54 @@ offset (optional):: Offset the result set by X number of results.
             "originatorTransactionType": "CARD_PAYMENT_FEE",
             "originatingTransactionUuid": "f4aa28a0-8c51-11e5-9b2e-94a4ee5f3593"
         }
-        ...
-```
+    ]
+}
+
+----
+--
++
+
+
+*Link transactions with card purchases.*
 
-### Link Finance Transactions With Card Purchases
 Finance transactions of types `CARD_PAYMENT`, `CARD_PAYMENT_FEE`, `CARD_REFUND` and `CARD_PAYMENT_FEE_REFUND` can be linked using the field `originatingTransactionUuid` to a card purchase using the `cardPaymentUUID1` field of the payment if using the Purchase API v1 or the `uuid` field of the payment if using the Purchase API v2.
 
 In the case of other transaction types, the value of the `originatingTransactionUuid` is not linkable to a specific card purchase and is not useful for external integrators.
 
-### Payouts
-`GET /organizations/us/payout-info`
+'''
+
+====
+
+=== Payouts
+`GET /organizations/{organizationUUID}/payout-info`
+
+[%collapsible]
+====
+**Parameters:**
+
+[grid="none", frame="none" cols="20%,10%,10%,10%,50%"]
+|===
+|Name|Type|In|Required/Optional|Description
+
+|`organizationUUID`|_string_|_path_|*_required_*|Organization identifier as UUID or `self to denote that the organization should be derived from the authenticated user.
```

Comment:
As this info doesn't require special styling, I'd think regular styling without bold and italic is good enough.
The same comment goes to similar occurrences in the reference tables.